scripts/testing/heatmap.py

```diff
@@ -0,0 +1,625 @@
+#! /usr/bin/env python
```

Comment:
Please update the header here as a comment that indicates where you retrieved this script (to give credit).